static/js/lib/queries/auth.js

```diff
@@ -0,0 +1,21 @@
+// @flow
+import { FLOW_LOGIN } from "../auth"
+
+import type { RequestConfig } from "redux-query"
+
+export default {
+  loginEmailMutation: (email: string, next: ?string): RequestConfig => ({
```

Comment:
Nit: I think the redux-query naming conventions would call these objects ["query configs"](https://github.com/amplitude/redux-query#query-configs), so I think these should be called `loginEmailQuery`/`loginPasswordQuery`, `loginEmailConfig`/`loginPasswordConfig`, or `loginEmailPostQuery`/`loginPasswordPostQuery`

Reply:
`redux-query` [examples](https://github.com/amplitude/redux-query/blob/master/site/src/demos/mutations/client.js.txt#L74) use `nameMutation` for mutations and `$nameRequest` in order to distinguish between the two types of queries these can be, so I've stuck with that naming convention here.